README.md

```diff
@@ -12,7 +12,7 @@ In your `package.json`:
 {
   "main": "dist/index.js",
   "module": "dist/index-esm.js",
-  "typings": "dist/index.d.ts",
+  "typings": "dist/src/index.d.ts",
```

Comment:
I'm a little confused about this - is the build command creating a `src` directory inside of `dist`?

Reply:
Thanks for pushing towards a better solution! Once we configured `tsconfig.types.json` properly, this issue went away.